sdks/python/src/opik/jsonable_encoder.py

```diff
@@ -55,13 +53,18 @@ def jsonable_encoder(obj: Any) -> Any:
                 encoded_list.append(jsonable_encoder(item))
             return encoded_list
 
-        if isinstance(obj, np.ndarray):
-            return jsonable_encoder(obj.tolist())
-
         for type_, encoder in _ENCODER_EXTENSIONS:
             if isinstance(obj, type_):
                 return jsonable_encoder(encoder(obj))
 
+        try:
```

Comment:
Let's import numpy at the top of the module 
```python
try:
    import numpy
except ImportError:
    numpy = None
```
and inside encoder function we'll have
```python
if numpy is not None:
    ...
```

There is not a big difference except for the fact that with the import inside this function, it will work significantly longer when called the first time. And I'd like to avoid this effect for such kind of functions.